src/DB_Command.php

```diff
@@ -456,6 +456,116 @@ public function tables( $args, $assoc_args ) {
 		}
 	}
 
+	/**
+	 * Display the database and table sizes.
+	 *
+	 * Displays the database and table sizes for `DB_NAME` specified in wp-config.php.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--db-only]
+	 * : Only display the database size without including the tables.
+	 *
+	 * [--format]
+	 * : table, csv, json, bytes
+	 * ---
+	 * default: table
+	 * options:
+	 *   - table
+	 *   - csv
+	 *   - json
+	 *   - bytes, only works when combined with --db-only
```

Comment:
I think this might work better as a `--size_format=<format>` argument that accepts one of `b`, `kb`, or `mb`. The default could be `kb`.